packages/astroport-governance/src/assembly.rs

```diff
@@ -201,6 +245,59 @@ pub struct Proposal {
     pub deposit_amount: Uint128,
 }
 
+impl Proposal {
+    pub fn validate(&self, whitelisted_links: Vec<String>) -> StdResult<()> {
+        // Title validation
+        if self.title.len() < MIN_TITLE_LENGTH {
+            return Err(StdError::generic_err("Title too short!"));
+        }
+
+        if self.title.len() > MAX_TITLE_LENGTH {
+            return Err(StdError::generic_err("Title too long!"));
+        }
+        if !self.description.chars().all(char::is_alphanumeric) {
```

Comment:
We should validate title in the same way.

Reply:
Changed.